pepper-apis/docs/specification/src/endpoints/studies.audit.yml

```diff
@@ -0,0 +1,96 @@
+post:
+  operationId: studyAudit
+  tags:
+    - Audit
+  summary: StudyAudit
+  description: |
+    Search and filter the log of the study.
+  parameters:
+    - $ref: '../pepper.yml#/components/parameters/studyGuid'
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            actionType:
+              description: the action
+              nullable: true
+              type: array
+              items:
+                type: string
+                enum:
+                  - CREATE
+                  - READ
+                  - UPDATE
+                  - DELETE
+                  - EXPORT
+            entityType:
+              description: the entity
+              nullable: true
+              type: array
+              items:
+                type: string
+                enum:
+                  - USER
+                  - ACTIVITY
+                  - INSTANCE
+                  - ANSWER
+            dateRange:
+              description: the date range to filter the log
+              nullable: true
+              type: object
+              $ref: '../pepper.yml#/components/schemas/DateRange'
+            userEmail:
+              description: the subject user's e-mails
+              nullable: true
+              type: array
+              items:
+                type: string
+            userGuid:
+              description: the subject user's GUIDs (must be specified in case entity type was set to User)
+              nullable: true
+              type: array
+              items:
+                type: string
+            activityCode:
+              description: the activity code (must be specified in case entity type was set to Activity)
+              nullable: true
+              type: array
+              items:
+                type: string
+            questionStableId:
+              description: the question stable id (must be specified in case entity type was set to Answer)
+              nullable: true
+              type: array
+              items:
+                type: string
+            limitation:
```

Comment:
This is for pagination purpose, right ?

Reply:
Right. I assume we would like to have something like infinite scroll (or pagination (from BE POV it's the same)), so I just propose to expose the limit constraint